include/caffe/util/mkl_alternate.hpp

```diff
@@ -33,6 +33,7 @@ extern "C" {
 
 DEFINE_VSL_UNARY_FUNC(Sqr, y[i] = a[i] * a[i]);
 DEFINE_VSL_UNARY_FUNC(Exp, y[i] = exp(a[i]));
+DEFINE_VSL_UNARY_FUNC(Ln, y[i] = log(a[i]));
```

Comment:
I kinda prefer the name `log` here, sticking with the name of the elementwise call, but either one is okay...